tests/platform_tests/test_platform_info.py

```diff
@@ -209,8 +211,7 @@ def turn_all_outlets_on(pdu_ctrl):
     for outlet in all_outlet_status:
         if not outlet["outlet_on"]:
             pdu_ctrl.turn_on_outlet(outlet)
-            time.sleep(5)
```

Comment:
Don't need to change here.

Reply:
It was adjusted in this PR since it was adjusted in the previous PR to increase the delay. I will remove this change though since it looks like it will not have any effect on test results.